### ISSUE TITLE
Attemt to deploy contract hang validator.

### ISSUE THREAD

**Issue by matsuro-hadouken** [regen-network/testnets#857]
Messing around contract deployment

After running this ( not sure if this even have any sense, just experiment )

```
NODE="http://127.0.0.1:26657"

regen tx wasm store cw20_base.wasm \
  --chain-id "aplikigo-1" \
  --gas auto \
  --from posfreak -y \
  --memo "PoSFreak" \
  --node "$NODE"
```

validator hang and RPC doesn't respond anymore. 
Nothing in a log, it just stop producing any output.

`systemctl restart cosmovisor` solved the issue.

_Can be related to `--gas auto` , just an idea._

**Comment by LexPrime** [comment]
Same problem. Got a dead node that had to be reloaded and a bunch of skipped blocks after using --gas auto in regen tx wasm store

**Comment by armyids** [comment]
Tried several times but the same result. If someone already reproduced this issue and post the solution here, that would be great.

**Comment by ryanchristo** [comment]
CosmWasm is no longer included in the latest version of regen ledger (v5.0). We may consider adding cosmwasm when there is a version that is compatible with the latest cosmos sdk version but we encourage cosmwasm contract developers to leverage interchain accounts to interact with regen ledger features via IBC.
